docs/AdminGuide.md

```diff
@@ -108,3 +108,54 @@ Example entry:
     "time": "2021-10-05 14:44:59",
 }
 ```
+
+## Update the expiry date of mismatches
+
+1. SSH into toolforge.
+
+    ```bash
+    ssh <your_username>@login.toolforge.org
+    ```
+
+1. The `become` commands allows you to sudo to another user while retaining your personalized environment.
+
+    ```bash
+    become mismatch-finder
+    ```
+
+1. Get into the database using the toolforge alias for accessing Mariadb 
+    ```bash
+    sql tools
+    ```
+
+1. Use the Wikidata mismatch finder database
+
+    ```sql
+    USE s54796__mismatch_finder; 
+    ```
+
+1. Now inside the database we can perform the update operation. 
+
+    ```sql
+    UPDATE import_meta SET expires='<date_of_expiration>' where id = <id_of_entry_to_update>; 
```

Comment:
Shall we maybe advice users to do this in a transaction? Otherwise for example a forgotten `WHERE` can hurt pretty bad.

Reply:
What would be the difference @mariushoch ? I am familiar with transactions.

Reply:
Changes you do in a transaction can be rolled back, so if you for example run `UPDATE import_meta SET expires='<date_of_expiration>';` (missing `WHERE` clause, will set the new expiration date for ALL imports), you can just issue a `ROLLBACK` to revert your mistake. Otherwise the only option would be to restore the old contents from a backup.

Reply:
that sounds definitely like a good idea. so the new command would be like this?

```sql
START TRANSACTION;
UPDATE import_meta SET expires='<date_of_expiration>' where id = <id_of_entry_to_update>; 
COMMIT;
```

Reply:
otherwise you can add a suggestion to the PR and i commit it. @mariushoch

Reply:
Yes, that looks good to me :)

Reply:
Done!